addons/website_sale/static/src/js/website_sale_dashboard/website_sale_dashboard.js

```diff
@@ -10,10 +10,17 @@ export class WebsiteSaleDashboard extends Component {
 	setup() {
 		this.state = useState({
 			eCommerceData: {},
-			selectedFilter: DATE_OPTIONS[0],
+			selectedDateFilter: DATE_OPTIONS[0],
+			selectedFilter: null,
```

Comment:
```suggestion
			selectedFilter: [],
```
should be declared as an empty array since you will be maintaining an array with this variable